uniffi/src/lib.rs

```diff
@@ -8,23 +8,24 @@
 //! component scaffolding in order to transfer data back and forth across the C-style FFI layer,
 //! as well as some utilities for testing the generated bindings.
 //!
-//! The key concept here is the [`ViaFfi`] trait, which must be implemented for any type that can
-//! be passed across the FFI, and which determines:
+//! The key concept here is the [`FfiConverter`] trait, which is responsible for converting between
+//! a Rust type and a low-level C-style type that can be passed across the FFI:
 //!
-//!  * How to [represent](ViaFfi::FfiType) values of that type in the low-level C-style type
+//!  * How to [represent](FfiConverter::FfiType) values of the rust type in the low-level C-style type
```

Comment:
```suggestion
//!  * How to [represent](FfiConverter::FfiType) values of the Rust type in the low-level C-style type
```

(I know I'm pedantic here)